tests/models/test_gpu.py

```diff
@@ -49,7 +49,7 @@ def test_multi_gpu_none_backend(tmpdir):
     tpipes.run_model_test(trainer_options, model)
 
 
-@pytest.mark.skipif(not torch.cuda.is_available(), reason="test requires GPU machine")
+@pytest.mark.skipif(torch.cuda.device_count() < 2, reason="test requires 2 GPUs")
```

Comment:
```suggestion
@pytest.mark.skipif(torch.cuda.device_count() < 2, reason="test requires multi-GPU machine")
```
just putting the same `reason` as it's there in other tests.

Reply:
awesome